backend/backend_test.go

```diff
@@ -0,0 +1,188 @@
+package backend_test
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/ausocean/cloud/backend"
```

Comment:
The main difference is whether or not the tests can call unexported methods. I also wonder whether it would make the compilation of binaries which use the backend package slightly smaller, although I don't know if it really matters.